ui/about.go

```diff
@@ -30,6 +46,8 @@ func (h *handler) showAboutPage(w http.ResponseWriter, r *http.Request) {
 	view.Set("user", user)
 	view.Set("countUnread", h.store.CountUnreadEntries(user.ID))
 	view.Set("countErrorFeeds", h.store.CountUserFeedsWithErrors(user.ID))
+	view.Set("showGlobalConfigOptions", showGlobalConfigOptions)
+	view.Set("globalConfigOptions", globalConfigOptions)
```

Comment:
You can just call `config.Opts.SortedOptions()` here if you do the modification mentioned earlier and remove other modifications made in this file.